pkg/scheduler/algorithm/predicates/max_attachable_volume_predicate_test.go

```diff
@@ -361,25 +362,25 @@ func TestVolumeCountConflicts(t *testing.T) {
 			newPod:       onePVCPod(EBSVolumeFilterType),
 			existingPods: []*v1.Pod{oneVolPod, deletedPVCPod},
 			filterName:   EBSVolumeFilterType,
-			maxVols:      2,
+			maxVols:      1,
 			fits:         false,
-			test:         "pod with missing PVC is counted towards the PV limit",
+			test:         "missing PVC is not counted towards the PV limit",
 		},
 		{
 			newPod:       onePVCPod(EBSVolumeFilterType),
 			existingPods: []*v1.Pod{oneVolPod, deletedPVCPod},
 			filterName:   EBSVolumeFilterType,
-			maxVols:      3,
+			maxVols:      2,
 			fits:         true,
-			test:         "pod with missing PVC is counted towards the PV limit",
+			test:         "missing PVC is not counted towards the PV limit",
 		},
 		{
 			newPod:       onePVCPod(EBSVolumeFilterType),
 			existingPods: []*v1.Pod{oneVolPod, twoDeletedPVCPod},
 			filterName:   EBSVolumeFilterType,
-			maxVols:      3,
-			fits:         false,
-			test:         "pod with missing two PVCs is counted towards the PV limit twice",
+			maxVols:      2,
```

Comment:
I miss some tests with unbound PVC and matching / non-matching StorageClass.

Reply:
Added tests for non-matching + invalid StorageClasses. The other tests are covering the matching use cases.